### ISSUE TITLE
Collection - C0187 is coming into Aeon requests with no site association.

### ISSUE THREAD

**Issue by ccleeton** [pulibrary/pulfalight#707]
A good number of items in C0187 is coming into aeon with no site associate which is causing it to go into a different queue. I have attached a screen shot from the Aeon client, here of a few transactions I made where this occurred
![site assoc fitz](https://user-images.githubusercontent.com/9676616/109525671-c46f5d00-7a7f-11eb-91d4-9a8859594189.png)
.

**Comment by tpendragon** [comment]
Did some quick research into this before heading off to another ticket: Looks like the physloc_code is `hsvm`, which we don't have mapped to a physical location in our indexing. According to PULFA, that should map to the RBSC site.

**Comment by tpendragon** [comment]
I can't figure out how this works in the old site. Maybe you can help @sdellis ?

https://findingaids.princeton.edu/collections/C0187/c00003 definitely requests a Site of RBSC, but the only physloc code is hsvm. That's populated from here:

https://github.com/PrincetonUniversityLibrary/pulfa/blob/e8f383c4135948412fd0f0f88bfd73201f05e6a1/webapp/xsl/component2html.xsl#L359-L361

However, that function it calls pulls from 

https://github.com/PrincetonUniversityLibrary/pulfa/blob/e8f383c4135948412fd0f0f88bfd73201f05e6a1/webapp/xsl/lib.xsl#L40-L50

via

https://github.com/PrincetonUniversityLibrary/pulfa/blob/e8f383c4135948412fd0f0f88bfd73201f05e6a1/webapp/xsl/lib.xsl#L912-L915

There's no RBSC there for hsvm. I don't see any logic that like...sets a default value or anything.

I can make everything that's not mapped fall back to the RBSC site, I'd just like evidence that's the right behavior.

**Comment by sdellis** [comment]
@tpendragon 
TLDR; I don't think HSVM items are getting routed correctly in the old site either. You may have discovered a bug.

My two attempts at using the old 2.0 site to request items in this collection resulted in requests that did not have a Site value. I tried in another collection and I got a site value. What I suspect is that the lookup function doesn't find a value. I would expect the hidden input field that it produces to be rendered with a blank value. However, the input field seems to never get rendered at all. If you've found an example of an HSVM item that renders the Site, let me know.

**Comment by tpendragon** [comment]
@sdellis https://findingaids.princeton.edu/collections/C0187/c00003 does right?

**Comment by tpendragon** [comment]
@ccleeton has said that things should just default to RBSC, so I'm gonna do that to close this ticket.

**Comment by ccleeton** [comment]
@bess @tpendragon Just got around to testing this and its still not showing up with a site in aeon

**Comment by ccleeton** [comment]
Just kidding. I got one that didn't have a site then when I tried to re-do it, it was fixed
